website/templates/project/addon/widget.mako

```diff
@@ -1,5 +1,5 @@
-<div class="panel panel-default" name="${short_name}">
-
+% if complete or 'write' in user['permissions']:
```

Comment:
The 'write' in user['permissions'] is throwing me here. Can you explain why this block shows for all write users even if it's not complete?